1-3.md

```diff
@@ -81,13 +85,14 @@ Approximately 30 to 45 minutes.
 
 #### Teaching Ideas
 
-* Discuss how accessibility solutions are also enjoyed by many people regardless of their disability. Emphasize how many solutions originally thought for accessibility have become mainstream, such as the type-writer, telephone, punch cards, text to speech, email, and voice controls.
-* Promote discussion among students about which aspects of accessibility benefit them as well as people with disabilities. For instance, ask them which products they can use without looking at them or without pressing any button.
+* Discuss how accessibility features benefit many people in different situations, such as mobile phone users, older people, and people using different hardware. Explain that many features originally thought for accessibility have become mainstream, such as the type-writer, telephone, punch cards, text to speech, email, and voice controls.
```

Comment:
Think we should either use "features" rather than "solutions". I think this is also more consistent with Units 1 & 2 but please check consistency of terms throughout all units.